joern-cli/frontends/c2cpg/src/test/scala/io/joern/c2cpg/dataflow/DataFlowTests.scala

```diff
@@ -49,7 +49,7 @@ class DataFlowTests extends DataFlowCodeToCpgSuite {
 
     "find flows to `free`" in {
       val source = cpg.identifier
-      val sink   = cpg.call.name("free")
+      val sink   = cpg.call.name("free").argument(1)
```

Comment:
@max-leuthaeuser I believe the intention here wasn't to check that the return value of `free` is tainted by `p`. The semantics for `free` currently specify that `1 -> 1`. What do you think?